derive_builder/examples/builder_function_for_the_struct.rs

```diff
@@ -0,0 +1,24 @@
+// NOTE: generate fully expanded version with `cargo expand`.
+//
+//       cargo expand --example new_builder_function
+#![allow(dead_code)]
+
+#[macro_use]
+extern crate derive_builder;
+
+#[derive(Default, Builder, Debug)]
+#[builder(setter(into))]
+struct Channel {
+    token: i32,
+    special_info: i32,
+}
+
+fn main() {
+    // create a new `ChannelBuilder` using `builder` fn from the `Channel` struct
+    let ch = Channel::builder()
```

Comment:
Thanks for including an example with this!

Reply:
at least thats the only thing good in this PR 😅